back-end/functions/test_auth.py

```diff
@@ -0,0 +1,33 @@
+import json
+import jwt
+from util.get_secret import get_secret
+
+SECRET_KEY = get_secret("auth", "jwt_secret_key")
+
+def is_jwt_token_valid(jwtToken):
+    try:
+        jwt.decode(jwtToken, key=SECRET_KEY, algorithms=["HS256"])
+        return True
+    except Exception:
+        return False
```

Comment:
Same thought as with `create_jwt_token()` in `auth.py`. Maybe `is_jwt_token_valid()` and `create_jwt_token()` can live in the same file, and those methods can be imported where needed?

Reply:
yea I agree, I'm still researching a good way to organize files in python sam project

Reply:
should be fixed now !